libs/search-widget/src/core/api.ts

```diff
@@ -57,6 +59,9 @@ export const initNuclia = (options: NucliaOptions, state: KBStates, widgetOption
   if (widgetOptions.features?.useSynonyms) {
     SEARCH_OPTIONS.with_synonyms = true;
   }
+  if (widgetOptions.features?.citations) {
```

Comment:
Maybe it could be problematic in the widget generator page.
if the citation checkbox is disabled, it seems that the `CITATIONS` variable would remain enabled.